samples/04_gis_analysts_data_scientists/information_extraction_from_cheshire_fire_incident_reports_using_mistral_language_model.ipynb

```diff
@@ -0,0 +1,1273 @@
+{
```

Comment:
<ul><li>Follow these steps to download and install the Mistral model backbone:</li></ul><ol><li class="ql-indent-1">Download the&nbsp;<a href="https://esri.maps.arcgis.com/home/item.html?id=969d2fc57c834295af1a4a42cfd51f68" target="_blank">Mistral Model Backbone</a>.</li><li class="ql-indent-1">Extract the downloaded zip file.</li><li class="ql-indent-1">Open the Anaconda Prompt and navigate to the folder that contains&nbsp;<code>arcgis_mistral_backbone-1.0.0-py_0.tar.bz2</code>.</li><li>Run the following command:</li></ol><ul><li class="ql-indent-1"><code>conda install --offline arcgis_mistral_backbone-1.0.0-py_0.tar.bz2</code></li></ul><blockquote><br></blockquote><blockquote>Will it not be redundant? As the detailed instruction is already present in the linked page</blockquote>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/2115/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>